functions.php

```diff
@@ -264,18 +265,75 @@ function register_more_blocks(): void
                     // For example, it could already access displayLayout from Query block to know how many columns are
                     // being rendered. If it then also knows the flex gap and container width, it should have all needed
                     // info to support a large amount of cases.
-                    [ 'sizes' => '(min-width: 1600px) 389px, (min-width: 1200px) 335px, (min-width: 1000px) 281px, (min-width: 780px) 209px, (min-width: 580px) 516px, calc(100vw - 24px)' ] // phpcs:ignore Generic.Files.LineLength.MaxExceeded
+                    [ 'sizes' => '(min-width: 1600px) 389px, (min-width: 1200px) 335px, (min-width: 1000px) 281px, (min-width: 780px) 209px, (min-width: 580px) 516px, calc(100vw - 24px)' ]
                 );
 
                 return "<a href='$post_link'>$featured_image</a>";
             },
             'uses_context' => [ 'postId' ],
         ]
     );
+    // Block displays related posts using the Query Loop block
+    register_block_type(
+        'p4/related-posts',
+        [
+            'attributes' => [
+                'queryAttributes' => [
```

Comment:
Is it not `query_attributes`? If I look [here](https://github.com/greenpeace/planet4-master-theme/pull/2370/files#diff-8e039a17b2aadb723e862bced701672b6e20a79e19ca6426b49084fc76d52624R96)